crates/gitbutler-core/src/virtual_branches/virtual.rs

```diff
@@ -1687,6 +1688,84 @@ fn get_non_applied_status(
         .collect::<Result<Vec<_>>>()
 }
 
+fn compute_merge_base(
+    project_repository: &project_repository::Repository,
+    integration_commit: &git::Oid,
```

Comment:
This type is `Copy` and thus shouldn't be referenced, just like (now) `ProjectId`.